extensions/Template.md

```diff
@@ -10,7 +10,7 @@ TODO: Draft or Stable
 
 ## Dependencies
 
-Written against the glTF draft 1.0 spec.
+Written against the glTF draft 2.0 spec.
```

Comment:
Probably should remove the `draft` also. :smirk:

```suggestion
Written against the glTF 2.0 spec.
```

Reply:
Haha, yes, that makes sense. :-)